src/renderer/core/api/api-service.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Disabling the rule for the whole file is quite broad. Instead one could just disable the rule for the affected line. e.g.
```
  postPing: (
    cohort: string,
    // eslint-disable-next-line @typescript-eslint/camelcase
    is_first: boolean
  ) => Promise<AjaxResponse | unknown>;
```
In this case I propose to replace the usages of `is_first` with `isFirst`. The name of the GET-parameter should stay `is_first`.